### ISSUE TITLE
Wrong size of legend symbols when using Geometry generator symbology

### ISSUE THREAD

**Issue by VxTedxV** [qgis/QGIS#36037]
I was not able to reopen a closed ticket : #22638

This issue remains in QGis 3.12.1 !

Tested with line geometry : in legend, the line is half screen long before we can read the label !

**Comment by gioman** [comment]
Duplicate of https://github.com/qgis/QGIS/issues/33755
